packages/office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlertPage.tsx

```diff
@@ -0,0 +1,121 @@
+              code={ ScreenReaderAlertRepeatExampleCode as string }
+            >
+              <ScreenReaderAlertRepeatExample />
+            </ExampleCard>
+            <ExampleCard
+              title='ScreenReaderAlert with Spinner'
+              code={ ScreenReaderAlertSpinnerExampleCode as string }
+            >
+              <ScreenReaderAlertSpinnerExample />
+            </ExampleCard>
+            <ExampleCard
+              title='ScreenReaderAlert polite mode'
+              code={ ScreenReaderAlertAfterExampleCode as string }
+            >
+              <ScreenReaderAlertAfterExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
+          <PropertiesTableSet
+            sources={ [
+              require<string>('!raw-loader!office-ui-fabric-react/src/components/ScreenReaderAlert/ScreenReaderAlert.Props.ts')
+            ] }
+          />
+        }
+        overview={
+          <div>
+            <p>
+              ScreenReaderAlerts are used to read out the notifications on the page to the blind people with the help of screen readers like JAWS, Chrome VOX and NVDA. It is implemented by live region and overcomes some known issues about live region when using with React.
+            </p>
+            <p>
+              It renders an addition element for screen reader only, so using this component while the text/ui is visible will duplicate the text in the DOM. You should use vanilla live region attributes instead if you don't want this duplicate element.
+            </p>
+            <p>
+              This component has been tested through following combination of screen readers and browsers:
+            </p>
+            <ul>
+              <li>
+                1. <strong>Narrator in windows 10 Enterprise 14393.577</strong> with <strong>Edge 38.14393.0.0</strong>.
+              </li>
+              <li>
+                2. <strong>NVDA 2016.4</strong> with <strong>FireFox v50.1.0</strong>.
+              </li>
+              <li>
+                3. <strong>Chrome VOX v53</strong> with <strong>Chrome v54</strong>.
+                <br />
+                Note: ReadingMode.ReadAfterOtherContent option is currently not supported in ChromeVOX, text will be read immediately.
+              </li>
+              <li>
+                4. <strong>JAWS 18</strong> with <strong>IE 11</strong>.
+              </li>
+            </ul>
+
+            <p>
+              To try the demo in this page, you should at least using one kind of screen reader. E.g. JAWS, ChromeVOX, NVDA.
+              Better use the browser recommended by the screen reader you're using. E.g. IE11 with JAWS, Chrome with ChromeVOX, FireFox with NVDA.
+            </p>
+          </div>
+        }
+        bestPractices={
+          <div></div>
+        }
+        dos={
+          <div>
+            <ul>
+              <li>Use if it would be hard for blind people to know dynamic changes on the page.</li>
```

Comment:
_The docs you have say to use it where it would be hard for blind people to know dynamic changes on the page, but isn't that subjective?_
I should add a demo for this case, actually this is the most frequently used scenario in our projects. We notify user in many places: an item of a grid is deleted, user add an item, user reorder the position of two items, etc... I'll add the demo in next commit

_but would we expect users to manually use this component, or would we just use this in the notification bubble?_
They should manually use this, the example of spinner should demo the scenario that while an async call is executing, how to notify the different status. If they want to use it in the notification bubble, a helper function which insert a live region into DOM might be more suitable.

For the Dropdown and TextField, I'm not exactly sure what the scenario would be, but this component is mean to help common live region scenarios. I will add more concrete examples/demos. In fact in our projects, we're not using the vanilla live region code anymore which means for all the screen reader user notification, we're delegating to the ScreenReaderAlert component. This saves a lot time of copy-pasting code and testing across user agents.
The side-effect we're ignoring which might make it not suitable for TextField error message is that this component duplicate the text in DOM if the alert message is already a part of the visible UI. For TextField error message, using ScreenReaderAlert will cause the text rendered in DOM twice, once in the normal UI, and once in the ScreenReaderAlert  component. Still we're using this component instead of remembering various or implementations for making live region work. (E.g., TextField is using some DeleyedRender component to make live region work properly) In our code, we tried to avoid this kind of code and avoid the learning curve of various live region implementation for devs.
There might could have a promoted version which wrapper the children component and read the text content of the children components, but I don't know how to do that yet, probably something like DelayedRender could help.